### PR TITLE
New Unit Tests

### PR DESCRIPTION
**Pull request by yaolin902**
plan to test backend and frontend to identify any bugs before review

### CODE REVIEW

**Comment by yaolin902** [comment]
```
(venv) ➜  test git:(unit-testing) ✗ pytest -vvv
============================= test session starts ==============================
platform darwin -- Python 3.13.0, pytest-8.3.3, pluggy-1.5.0 -- /Users/mac/Pictures/MyInspectionBuddy/AwesomeProject/Backend/venv/bin/python3.13
cachedir: .pytest_cache
rootdir: /Users/mac/Pictures/MyInspectionBuddy/AwesomeProject/Backend/test
collected 11 items                                                             

test_api.py::test_search_fda PASSED                                      [  9%]
test_api.py::test_search_k510 PASSED                                     [ 18%]
test_api.py::test_search_cdph PASSED                                     [ 27%]
test_api.py::test_search_maude PASSED                                    [ 36%]
test_api.py::test_search_openhistorical PASSED                           [ 45%]
test_api.py::test_search_cabusinessentity PASSED                         [ 54%]
test_api.py::test_getcontacts PASSED                                     [ 63%]
test_api.py::test_getlicenses PASSED                                     [ 72%]
test_api.py::test_serpapi PASSED                                         [ 81%]
test_api.py::test_predict PASSED                                         [ 90%]
test_api.py::test_warningletter PASSED                                   [100%]

============================= 11 passed in 18.51s ==============================
```

**Comment by yaolin902** [comment]
Frontend testing requires:
`npm install --save-dev jest`
`npm install @testing-library/react-native`

File: AwesomeProject/Backend/main.py
Comment (argus1) [inline AwesomeProject/Backend/main.py:1]: change name of 'super secret key'

Comment (yaolin902) [reply AwesomeProject/Backend/main.py:1]: fixed

**Pull request merged**